### PR TITLE
add en locale

### PR DESCRIPTION
**Pull request by bicatcho11**
## Description

add en locale

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "popping_pills":"Popping pills..",
+      "healing":"Healing..",
+      "smoking_meth":"Smoking meth..",
+      "lighting_joint":"Lighting joint.."
+   },
+   "info":{
+      "minutes":"minutes",
+      "seconds":"seconds.."
+   }
+}
```
Comment (solareon) [inline locales/en.json]:
```suggestion
{
   "error": {
      "canceled": "Canceled..",
      "cruise_control_disabled": "Cruise control disabled",
      "cruise_control_unavailable": "Cruise control unavailable",
      "deleted_recording": "Deleted Recording!",
      "later_aligator": "Later aligator!"
   },
   "success": {
      "cruise_control_enabled": "Cruise control enabled",
      "started_recording": "Started Recording!",
      "saved_recording": "Saved Recording!"
   },
   "progress": {
      "eating": "Eating..",
      "drinking": "Drinking..",
      "drinking_liquor": "Drinking liquor..",
      "quick_sniff": "Quick sniff..",
      "smoking_crack": "Smoking crack..",
      "popping_pills": "Popping pills..",
      "healing": "Healing..",
      "smoking_meth": "Smoking meth..",
      "lighting_joint": "Lighting joint.."
   },
   "info": {
      "minutes": "minutes",
      "seconds": "seconds.."
   }
}
```
The spaces are to improve readability.

**Review by solareon — CHANGES_REQUESTED** [review]

File: locales/en.json (hunk 1 above)
Comment (TonybynMp4) [reply locales/en.json]: I might see wrong but did you remove all the `}` ?

Comment (solareon) [reply locales/en.json]: > I might see wrong but did you remove all the `}` ?

Nah it's just github doing github things.

**Review by solareon — APPROVED** [review]

**Review by BerkieBb — APPROVED** [review]

**Comment by bicatcho11** [comment]
wait to  approve

**Pull request merged**